requirements-dev.txt

```diff
@@ -1,11 +1,11 @@
 # everything the developer needs in addition to the production requirements
 
 # testing
-pytest==3.8
-pytest-flask==0.11
+pytest>=3.7.1
+pytest-flask
 pytest-cov
 mongomock<=3.12
-python-coveralls
+coveralls
```

Comment:
There is also coveralls-python. Reminds me, perhaps not without reason, of Monty Python. But, this seems official enough, and importantly to work so lets go with it!